keyboards/treadstone48/rev1/info.json

```diff
@@ -0,0 +1,182 @@
+[
```

Comment:
This file should be replaced, as it's not valid for its purpose of QMK Configurator support.

Here's a properly-formatted file to get you started. (Only supports `LAYOUT_base`.)

https://github.com/noroadsleft/qmk_firmware/blob/9ec608626156f362723064be990014be6bf65bbf/keyboards/treadstone48/rev1/info.json

For more on this, please see the following docs:

- [Configurator Support](https://docs.qmk.fm/#/reference_configurator_support)
- [info.json Format](https://docs.qmk.fm/#/reference_info_json)